glide.yaml

```diff
@@ -7,7 +7,7 @@ import:
 - package: github.com/crossdock/crossdock-go
   version: master
 - package: github.com/gogo/protobuf
-  version: ~0.5
+  version: ^1
```

Comment:
Advancing from ~0.5 to ^1 is a breaking change. Was the predicate mistakenly set at ~0.5 before? cc @peter-edge @abhinav

Reply:
There are no breaking changes between 0.5 and 1.0. We can change this to `>= 0.5, < 2.0` if we want to be extra prudent.